ibm/resource_ibm_pi_volume_attach.go

```diff
@@ -20,52 +20,41 @@ func resourceIBMPIVolumeAttach() *schema.Resource {
 	return &schema.Resource{
 		CreateContext: resourceIBMPIVolumeAttachCreate,
 		ReadContext:   resourceIBMPIVolumeAttachRead,
-		UpdateContext: resourceIBMPIVolumeAttachUpdate,
 		DeleteContext: resourceIBMPIVolumeAttachDelete,
 		Importer:      &schema.ResourceImporter{},
 
 		Timeouts: &schema.ResourceTimeout{
-			Create: schema.DefaultTimeout(60 * time.Minute),
-			Delete: schema.DefaultTimeout(60 * time.Minute),
+			Create: schema.DefaultTimeout(15 * time.Minute),
+			Delete: schema.DefaultTimeout(15 * time.Minute),
 		},
 
 		Schema: map[string]*schema.Schema{
 
-			"volumeattachid": {
-				Type:        schema.TypeString,
-				Computed:    true,
-				ForceNew:    true,
-				Optional:    true,
-				Description: "Volume attachment ID",
-			},
-
 			helpers.PICloudInstanceId: {
 				Type:        schema.TypeString,
 				Required:    true,
+				ForceNew:    true,
 				Description: " Cloud Instance ID - This is the service_instance_id.",
 			},
 
 			helpers.PIVolumeAttachName: {
 				Type:        schema.TypeString,
 				Required:    true,
+				ForceNew:    true,
```

Comment:
Instead of requesting name from the user for volume and instance...Can we take the id of them so that we can elimintate using d.Get in Read Method
```
servername := d.Get(helpers.PIInstanceName).(string)
	name := d.Get(helpers.PIVolumeAttachName).(string)
```


Can volClient.Attach(servername, name)  take the id instead of name?

Reply:
yes, it can be done, will change it to id.

Reply:
changed name to id